### PR TITLE
Add intro section for tutorial

### PR DESCRIPTION
**Pull request by klappradla**
This adds the intro section for the tutorial, also addressing #10. It includes:
- a brief explanation what the tutorial is about
- a brief explanation what "snake" the game is
- a quick walk through for installing all prerequisites _(Elixir, the `scenic_new` tasks)_

Rather than diving into the details on how to install Elixir or the Open GL libraries for the `scenic_new` tasks, I just pointed to the installation guides of the respective things on the internet. Given we'll have an experienced group, an installation party the day before, an intro to Elixir at the beginning of the workshop and coaches around, I feel like this is a) good enough for the tutorial and b) easier to maintain.

In case we need a separate, more detailed guide for setting up a development environment, I would put this into a tutorial for the installation party.

Feedback welcome 👋

### CODE REVIEW

File: docs/GUIDES.md [hunk 1]
```diff
@@ -25,22 +25,44 @@ A first collection things we need to explain (very incomplete):
...
+
+Once installed, check the version by running this command in your terminal:
+
+    $ elixir --version
+
+If your version is lower than `1.9.x`, either update to a more recent version or ask a coach for help.
+
+Next we'll install some `mix` tasks to help us build a `Scenic` application. [mix](https://hexdocs.pm/mix/Mix.html) is a tool that comes with Elixir to help developing apps and manage their dependencies. It it similar to e.g. `npm` in JavaScript (don't worry if you don't know that).
+
+The tasks in the `scenic_new` archive have some prerequisites. Head to the [instructions](https://github.com/boydm/scenic_new#install-prerequisites) for your operating system and install them first.and find the instructions for your operating system.
```
Comment (ggpasqualino) [inline docs/GUIDES.md]: Is this maybe extra?
```suggestion
The tasks in the `scenic_new` archive have some prerequisites. Head to the [instructions](https://github.com/boydm/scenic_new#install-prerequisites) for your operating system and install them first.
```

File: docs/GUIDES.md [hunk 2]
```diff
@@ -25,22 +25,44 @@ A first collection things we need to explain (very incomplete):
...
+
+We first need to make sure we have Elixir installed on our computer. [Installing Elixir](https://elixir-lang.org/install.html) has installation instructions for all tastes and operating systems.
+
+Once installed, check the version by running this command in your terminal:
+
+    $ elixir --version
+
+If your version is lower than `1.9.x`, either update to a more recent version or ask a coach for help.
+
+Next we'll install some `mix` tasks to help us build a `Scenic` application. [mix](https://hexdocs.pm/mix/Mix.html) is a tool that comes with Elixir to help developing apps and manage their dependencies. It it similar to e.g. `npm` in JavaScript (don't worry if you don't know that).
```
Comment (ggpasqualino) [inline docs/GUIDES.md]:
```suggestion
Next, we'll install some `mix` tasks to help us build a `Scenic` application. [mix](https://hexdocs.pm/mix/Mix.html) is a tool that comes with Elixir to help developing apps and manage their dependencies. It is similar to e.g. `npm` in JavaScript (don't worry if you don't know that).
```

File: docs/GUIDES.md [hunk 3]
```diff
@@ -25,22 +25,44 @@ A first collection things we need to explain (very incomplete):
...
+
+The tasks in the `scenic_new` archive have some prerequisites. Head to the [instructions](https://github.com/boydm/scenic_new#install-prerequisites) for your operating system and install them first.and find the instructions for your operating system.
+
+The `scenic_new` tasks require some additional libraries installed on your computer. See the [install prerequisites](https://github.com/boydm/scenic_new#install-prerequisites) for installation instructions for your operating system.
```
Comment (ggpasqualino) [inline docs/GUIDES.md]: This paragraph seems to achieve the same as the previous one, maybe we can actually remove the previous one.

File: docs/GUIDES.md [hunk 4]
```diff
@@ -25,22 +25,44 @@ A first collection things we need to explain (very incomplete):
...
+    $ mix archive.install hex scenic_new
+
+Now we are prepared to start building our Scenic application!
+
+Navigate into your personal projects directory _(or wherever you want to keep the files for the tutorial)_ and then let's get started 🚀
 
 ### 1. Create a scenic app
 
-First off we need to create a new Scenic application using our previously installed `scenic_new` mix task. We are going to build a snake game, so let's call our project `snake`. The `scenic_new` package gives us the handy `scenic.new` task which can be used to bootstrap a new Scenic application.
+First off we'll to create a new Scenic application using our previously installed `scenic_new` mix task. We are building a snake game, so let's call our project `snake`. The `scenic_new` package gives us the handy `scenic.new` task which can be used to bootstrap a new Scenic application.
```
Comment (ggpasqualino) [inline docs/GUIDES.md]:
```suggestion
First off we'll create a new Scenic application using our previously installed `scenic_new` mix task. We are building a snake game, so let's call our project `snake`. The `scenic_new` package gives us the handy `scenic.new` task which can be used to bootstrap a new Scenic application.
```

File: docs/GUIDES.md [hunk 5]
```diff
@@ -25,22 +25,44 @@ A first collection things we need to explain (very incomplete):
...
+We want to learn more about the [Elixir](https://elixir-lang.org) programming language by building a fun game together: **snake** - a cult game from mobile phones in the 1990s 🕹
 
-- explain what Scenic is
-- explain what `mix` is
-  - run `mix help` to see what commands are available and what they do
-  - use mix as a task runner here (compare to npm, yarn)
-    - rake is only a task runner
-- install the `scenic_new` mix task
-  - install an archive locally
+![snake game on a nokia phone](https://media.giphy.com/media/ZYOybCzZvpcY0/giphy.gif)
```
Comment (ggpasqualino) [inline docs/GUIDES.md]: :purple_heart:

File: docs/GUIDES.md (hunk 3 above)
Comment (nickgnd) [reply docs/GUIDES.md]: I found the last 2 paragraphs a little bit disconnected.
What about something like that?


```
# 👇 previous paragraph
Next, we'll install some `mix` tasks to help us build a `Scenic` application. [mix](https://hexdocs.pm/mix/Mix.html) is a tool that comes with Elixir to help developing apps and manage their dependencies. It is similar to e.g. `npm` in JavaScript (don't worry if you don't know that)

# 👇 current
Specifically, we will use the `scenic_new` tasks which require some additional libraries installed on your computer. ...
```

Comment (klappradla) [reply docs/GUIDES.md]: Bellissima 💚

**Comment by LuisaAPF** [comment]
I don't know whether this should be on the guides, but `make` and  `gcc` are also required to compile `scenic`. I usually run my projects inside Docker and these packages are usually provided by the images, so I didn`t have them installed locally.

**Comment by ggpasqualino** [comment]
@LuisaAPF, we decided to leave the installation details out and write down here only the links to the specific installation guides. This way is easier to maintain this tutorial and if we decide later on we need a more detailed installation guide for the installation party, then we can make that separately.

**Comment by LuisaAPF** [comment]
The guide for `scenic.new` also doesn't explicitly state that you need these packages. I guess it's just taken for granted that the developers will have them already. So I think it's good to keep that in mind, in case someone else has a problem when compiling scenic.

**Comment by klappradla** [comment]
Awesome @LuisaAPF, thanx for bringing this up 💜 Super helpful to have someone go through the steps of the tutorial with a "fresh" computer.

As @ggpasqualino says, we try to for now keep the details for installing things outside of the guide and just point to the usually more detailed, better maintained at more frequently tested external resources (e.g. for installing Elixir...).

But as you mention that these two things are actually **missing** from the external documentation, we should somehow keep track of that. I'll add it to the guide, either directly in the tutorial or in a separate `TROUBLESHOOTING.md` document that we'll link to from the guide.

In addition to that, you could also open an issue / a PR on [scenic_new](https://github.com/boydm/scenic_new).

What do you think?

**Comment by LuisaAPF** [comment]
Hey @klappradla happy to help :blush: . I'll make a suggestion to the maintainers of scenic_new to add this to the guides.

**Comment by jvf** [comment]
Awesome stuff, thanks for all the work!

❤️ 💚 💙 💛 💜

**Pull request merged**